devices/tuya.js

```diff
@@ -1658,7 +1658,7 @@ module.exports = [
         toZigbee: [tz.tuya_radar_sensor],
         exposes: [
             e.illuminance_lux(), e.presence(),
-            exposes.enum('motion', ea.STATE, ['true', 'false']).withDescription('moving inside the range of the sensor'),
+            exposes.binary('motion', ea.STATE, ['true', 'false']).withDescription('moving inside the range of the sensor'),
```

Comment:
Please update the converter to expose the bool as a bool and not a string and change this to `exposes.binary('motion', ea.STATE, [true, false])`

Reply:
Hi Koenkk,

thanks for your help. The converter allready expose it as bool. So I've made only the mistake inside /devices/tuya.js.

Reply:
I think the value is a string not a bool, please update https://github.com/Koenkk/zigbee-herdsman-converters/blob/d02a164ae94a0c51adca8ea31ff7aae141abe637/converters/fromZigbee.js#L7314 and to the same for `presence`